langchain/callbacks/tracers/langchain.py

```diff
@@ -72,6 +119,8 @@ def __init__(
         self.example_id = example_id
         self.session_name = session_name or os.getenv("LANGCHAIN_SESSION", "default")
         self.session_extra = session_extra
+        # set max_workers to 1 to process tasks in order
```

Comment:
Why does order matter? Can we rely on every user of the api being as strict about order as we are being here?

Reply:
It'll fail if a child run gets posted before a parent run at least, right? (parent_run_id not found)